sycl/include/sycl/accessor.hpp

```diff
@@ -832,17 +848,12 @@ class __SYCL_SPECIAL_CLASS accessor :
 
   template <int Dims = Dimensions> size_t getLinearIndex(id<Dims> Id) const {
 
-#ifdef __SYCL_DEVICE_ONLY__
-    // Pointer is already adjusted for 1D case.
-    if (Dimensions == 1)
-      return Id[0];
-#endif // __SYCL_DEVICE_ONLY__
-
     size_t Result = 0;
-    // Unroll the following loop for both host and device code
-    __SYCL_UNROLL(3)
```

Comment:
We probably don't need to keep `__SYCL_UNROLL` around then.

Reply:
Do you mean the definition of it?

Reply:
Yes, that's what I meant.